vmsgen.py

```diff
@@ -1,3 +1,6 @@
+# Copyright 2016-2020 VMware, Inc.
```

Comment:
so, this is the only file which existed in 2019? all the other files have only 2020.

Reply:
Yes, along with test_vmsgen.py, which was broken down to different test files and is currently non-existent. 

(I am not counting any *.md files)